docs/tianocore_build.md

```diff
@@ -49,6 +49,8 @@ PC Engines apu2 platform. Payload is supported since v4.9.0.7 release.
   - In `Add a payload` choose *Tianocore coreboot payload package*
   - Select tianocore build type release
   - In `Secondary Payloads` disable all options
+  - Make sure that `Insert a commit’s SHA-1 or a branch name` 
```

Comment:
Please add information that it is the Tianocore revision setting. Additionally, you could add a screenshot from Payloads menu?